.docs/content/2.usage/use-armonik-api-in-an-angular-app.md

```diff
@@ -0,0 +1,549 @@
+::
+
+Then, we can create a method to call the `listPartitions` service:
+
+```typescript [partitions-grpc.service.ts]
+import { Injectable, inject } from '@angular/core';
+import { ListPartitionsRequest, ListPartitionsResponse, PartitionsClient } from '@aneoconsultingfr/armonik.api.angular';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class PartitionsGrpcService {
+  readonly #client = inject(PartitionsClient);
+
+  list$(): Observable<ListPartitionsResponse> {
+    const options = new ListPartitionsRequest();
+
+    return this.#client.listPartitions(options);
+  }
+}
+```
+
+_Voilà!_ We have created our first gRPC service! Now, we can use it in our component.
+
+### Create the component
+
+::alert{type="info"}
+In order to simplify this guide, will use the `AppComponent` to call the service. In a real application, you should create a dedicated component using the router.
+::
+
+For this guide, we will display the result in a list, having a loading indicator, a button to refresh the list and an error message if the call failed (printed in the console).
+
+#### Display partitions
+
+First, let's create the list using HTML:
+
+```html [app.component.ts]
+<ul>
+  <li *ngFor="let partition of partitions; trackBy:trackByPartition">
+    {{ partition.id }}
+  </li>
+</ul>
+```
+
+::alert{type="info"}
+You must add this code in the `template` property of the `@Component` decorator.
+::
+
+Then, we must update our component with some properties and methods:
+
+```typescript [app.component.ts]
+import { PartitionRaw } from '@aneoconsultingfr/armonik.api.angular';
+
+@Component({
+  // ...
+})
+export class AppComponent {
+  // ...
+  partitions: PartitionRaw.AsObject[] = [];
+
+  trackByPartition(_index_: number, partition: PartitionRaw.AsObject): string {
+    return partition.id;
+  }
+}
+```
+
+Finally., we must import `ngFor` in the `imports` property of the `@Component` decorator:
+
+```typescript [app.component.ts]
+import { Component } from '@angular/core';
+import { NgFor } from '@angular/common';
+
+@Component({
+  // ...
+  imports: [
+    NgFor
+  ]
+})
+```
+
+#### Display the loading indicator
+
+We will use the `*ngIf` directive to display the loading indicator:
+
+```html [app.component.ts]
+<div *ngIf="loading">
+  Loading...
+</div>
+```
+
+::alert{type="info"}
+You must add this code in the `template` property of the `@Component` decorator.
+::
+
+Then, we must update our component with some properties and methods:
+
+```typescript [app.component.ts]
+@Component({
+  // ...
+})
+export class AppComponent {
+  // ...
+  loading = true;
+}
+```
+
+::alert{type="info"}
+By default, the loading indicator will be displayed because data will be fetch on page initialization. We will hide it when the call is done.
```

Comment:
By default, the loading indicator will be displayed because data will be fetched on page initialization. We will hide it when the call is done.